### DISCUSSION TITLE
Restrict TileLayer extents

### DISCUSSION THREAD

**Discussion by RhythmOfTension** [pelicanmapping/osgearth#2208] [Support Q&A]
Is there a way to restrict TileLayer's extents? For example, i have GDALImageLayer from datasource which covers whole earth, but i need osgEarth to render only some region, can i do that?

Upd: i guess my question not only about TileLayer, but VisibleLayer (for example FeatureModelLayer)

**Comment by gwaldron** [comment]
No, there is no way to do that in osgEarth. However since you are using a GDALImageLayer, you may be able to use a tool like `gdalbuildvrt` to create a GDAL VRT that restricts the extents of the image -- I would look into that.
